### DISCUSSION TITLE
Chromium browser renders the blank screen for the first time and causing behavioral issues in WPF application for the specific machine configuration.

### DISCUSSION THREAD

**Discussion by Padma321** [cefsharp/CefSharp#3928] [Q&A]
Hello Team,
We have been facing the Chromium web browser visual issue in WPF Based application and behavioral issues in specific machine configuration. We have been trying lot of fixes and upgraded the CEFSharp to the most recent version. However, the same issue is happening.

**Issue 1:**
The Chromium browser renders the blank screen. However, If the screen is resized, the screen is rendering. Once it's rendered, it continues to render the screen until the application is re-launched. It goes blank only for the first time loading.

**Issue 2:**
The action controls like WPF Button and WPF Text box are in disabled state after the screen is rendered. Also, The actions performed in disabled controls are visible while resizing the application.

**Note:** The issue is not observed in Dell Latitude 5400 laptop model. However, these issues have been observed in Latitude 5420 and 7420 model laptops. These issues are happening in latest version of CEFSharp package as well.

**Additional Info:**
OS: Windows 10
Model: Dell Latitude 5420 and 7420
Application: WPF application
CEFSharp version: 96.0.142.0

**Sample Code snippet:**
 <wpf:ChromiumWebBrowser
                Name="Browser"
				Height="auto"
                Width="1364"
                MouseDown="Browser_OnMouseDown"
                Address= "{Binding RelationshipTreeViewAddress}"
                FrameLoadStart="Browser_OnFrameLoadStart"/>

Can you help us resolving these issues and provide the solution ASAP?

**Comment by amaitland** [comment]
> WPF the browser stops redrawing when running on Intel Iris Xe Integrated GPU(11th Gen).
Intel have released an updated driver, see https://www.intel.com/content/www/us/en/support/articles/000058136/graphics/graphics-for-11th-generation-intel-processors.html See dotnet/wpf#3817 for workaround if you are unable to install the updated driver.

As per https://github.com/cefsharp/CefSharp/releases/tag/v96.0.141

**Reply by Padma321** [reply]
Hi Amaitland, I have used the above link and tried the workaround which is posted by Deiruch in the discussion forum (https://github.com/dotnet/wpf/issues/3817). This fixes the scaling issue in Chromium browser. Thank you for the workaround.

//work around a bug in old intel drivers: https://www.intel.com/content/www/us/en/support/articles/000058136/graphics/graphics-for-11th-generation-intel-processors.html

```c#
foreach (ManagementObject video in new ManagementObjectSearcher(new SelectQuery("Win32_VideoController")).Get())
if ((string)video["Name"] == "Intel(R) Iris(R) Xe Graphics" && string.CompareOrdinal((string)video["DriverVersion"], "30.0.100.9667") <= 0)
{
	System.Windows.Media.RenderOptions.ProcessRenderMode = System.Windows.Interop.RenderMode.SoftwareOnly;
	break;
}
```
